server/user_account/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.contrib.auth.models import AbstractBaseUser, BaseUserManager
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from rest_framework.authtoken.models import Token
+
+
+class CustomUser(AbstractBaseUser):
+    USERS = (
+        ('SA', 'System Admin'),
+        ('IM', 'Inventory Manager'),
+        ('SK', 'Stock Keeper'),
+    )
+    user_name = models.CharField(max_length=30, unique=True)
+    first_name = models.CharField(max_length=30)
+    last_name = models.CharField(max_length=30)
```

Comment:
Why a max length of 30?

Reply:
We can redefine them, but I didn't think that we would have names bigger than 30 characters, I can redefine it to what we decide.

Reply:
okay, maybe we can discuss with Ajay what he might expect, or we can do our own research to see the norm.

Reply:
[https://stackoverflow.com/questions/20958/list-of-standard-lengths-for-database-fields](https://stackoverflow.com/questions/20958/list-of-standard-lengths-for-database-fields)

Let's just use a safe 255 characters. I think that would be our safest choice